Formula/sdl_image.rb

```diff
@@ -12,14 +12,22 @@ class SdlImage < Formula
     sha256 "84272135304e7c475e8000a90af113127858273a4449ce6566805f5b4c2cb476" => :mavericks
   end
 
+  option :universal
+
   depends_on "pkg-config" => :build
   depends_on "sdl"
   depends_on "jpeg" => :recommended
   depends_on "libpng" => :recommended
   depends_on "libtiff" => :recommended
   depends_on "webp" => :recommended
 
-  option :universal
+  # Fix graphical glitching
+  # https://github.com/Homebrew/homebrew-python/issues/281
+  # https://trac.macports.org/ticket/37453
+  patch :p0 do
+    url "https://trac.macports.org/export/148198/trunk/dports/devel/libsdl_image/files/IMG_ImageIO.m.patch"
```

Comment:
Mind submitting this to `https://github.com/Homebrew/formula-patches` instead? Feel free to credit MacPorts still. 

We've just had past situations where Homebrew users haven't been able to install formula for 3+ days because MacPorts were having server problems, and it's hard to not feel like MacPort's servers shouldn't be a Homebrew dependency in any situation.